Library/Homebrew/api.rb

```diff
@@ -35,8 +35,8 @@ def self.fetch(endpoint)
       raise ArgumentError, "Invalid JSON file: #{Tty.underline}#{api_url}#{Tty.reset}"
     end
 
-    sig { params(endpoint: String, target: Pathname).returns([T.any(Array, Hash), T::Boolean]) }
-    def self.fetch_json_api_file(endpoint, target:)
+    sig { params(endpoint: String, target: Pathname, jws: T::Boolean).returns([T.any(Array, Hash), T::Boolean]) }
+    def self.fetch_json_api_file(endpoint, target:, jws: false)
```

Comment:
Might want to expand this, it's not going to be clear to most people what `jws` means in this context and a quick search didn't help here.